packages/tslint-config/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "description": "Wire TSLint config",
+  "license": "GPL-3.0",
+  "main": "tslint.json",
+  "name": "@wireapp/tslint-config",
+  "peerDependencies": {
+    "prettier": ">=1",
+    "tslint-config-prettier": ">=1",
+    "tslint-plugin-prettier": ">=2",
+    "tslint-react-hooks": ">=2",
+    "tslint-react": ">=4",
+    "tslint": ">=5"
+  },
+  "repository": "https://github.com/wireapp/wire-web-packages/tree/master/packages/tslint-config",
+  "version": "0.0.3"
```

Comment:
```suggestion
  "version": "0.0.1"
```

Reply:
I already published it and use it in this project :) It was easier than yarn link